bot/resources/tags/docstring.md

```diff
@@ -1,18 +1,16 @@
-A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that often used in file, classes, functions, etc. A docstring should have a clear explanation of exactly what the function does. You can also include descriptions of the function's parameter(s) and its return type, as shown below.
+A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string - always using triple quotes - that's placed at the top of files, classes and functions. A docstring should contain a clear explanation of what it's describing. You can also include descriptions of the subject's parameter(s) and what it returns, as shown below:
 ```py
-def greet(name, age) -> str:
-  """
-  Return a string that greets the given person, including their name and age.
+def greet(name: str, age: int) -> str:
+    """
+    Return a string that greets the given person, using their name and age.
 
-  :param name: The name to greet.
-  :type name: str
-  :param age: The age to display.
-  :type age: int
-  :return: String of the greeting.
-  """
-  return_string = f"Hello, {name} you are {age} years old!"
-  return return_string
+    :param name: The name of the person to greet.
+    :param age: The age of the person to greet.
+
+    :return: The greeting.
+    """
+    return f"Hello {name}, you are {age} years old!"
 ```
-You can get the docstring by using `.__doc__` attribute. For the last example you can get it through: `print(greet.__doc__)`.
+You can get the docstring by using the `.__doc__` attribute. For the last example, you can print it by doing this: `print(greet.__doc__)`.
```

Comment:
Would `inspect.getdoc` make more sense here? I'm not sure if the attribute is intended to be accessed directly, or something like getdoc which also cleans it up is a better alternative

Reply:
It's kind of like `dir()` vs. `.__dict__`.

Reply:
yea, good idea. Added in https://github.com/python-discord/bot/pull/1693/commits/39500b40ac2892cb469366103022bd854f7734b2

Reply:
> It's kind of like `dir()` vs. `.__dict__`.

inspect.getdoc Also clears indents, which is nice